CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 4.1.0
```

Comment:
No version bump required since everything in 4.0.0 is unreleased. Just move down into 4.0.0

Reply:
agreed. Done